djstripe/models/base.py

```diff
@@ -418,18 +419,33 @@ def _stripe_object_field_to_foreign_key(
 
         field_data = None
         field_name = field.name
-        raw_field_data = manipulated_data.get(field_name)
+        # raw_field_data = manipulated_data.get(field_name, "NA")
```

Comment:
Remove this line

Reply:
Good catch. I totally forgot to remove that.